packages/dialog/src/vaadin-dialog-resizable-mixin.js

```diff
@@ -75,6 +75,10 @@ export const DialogResizableMixin = (superClass) =>
         if (this.$.overlay.$.overlay.style.position !== 'absolute' || this.width || this.height) {
           this.$.overlay.setBounds(this._originalBounds);
         }
+
+        if (!this.$.overlay.hasAttribute('has-bounds-set')) {
+          this.$.overlay.setAttribute('has-bounds-set', '');
+        }
```

Comment:
Is the check necessary?

```suggestion
        this.$.overlay.setAttribute('has-bounds-set', '');
```

Reply:
Not really, let's remove it.